File: service/jvm-java-quarkus-reactive/src/main/java/com/ecobenchmark/controllers/addtask/TaskRequest.java
package com.ecobenchmark.controllers.addtask;

import io.quarkus.runtime.annotations.RegisterForReflection;

@RegisterForReflection
public class TaskRequest {

    private String name;

    private String description;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

}

File: service/php-symfony-apache2-dev/src/Controller/AddTask/TaskRequest.php
<?php

namespace App\Controller\AddTask;

use App\Entity\ListEntity;
use App\Entity\Task;

class TaskRequest
{
    public string $name;
    public string $description;

    function toTaskEntity(ListEntity $listEntity): Task
    {
        $task = new Task();
        $task->setList($listEntity);
        $task->setName($this->name);
        $task->setDescription($this->description);
        $task->setCreationDate(new \DateTime());

        return $task;
    }
}
